### DISCUSSION TITLE
loadPrevNext functionality missing since v9?

### DISCUSSION THREAD

**Discussion by Carnoisseur1979** [nolimits4web/swiper#6350] [Q&A] [answered]
In Swiper 9 the old lazy loading module was removed, and replaced with using native 'loading=lazy'. In the module, there were the options loadPrevNext and loadPrevNextAmount, which allowed you to tell Swiper to preload the next image and specify how many images would be preloaded. This is useful to avoid a loading delay when switching slides. Is there a way to get this functionality in Swiper 9? Thanks

**Comment by nolimits4web** [comment, marked as answer]
yes, it is missing in v9

**Reply by Carnoisseur1979** [reply]
Would you accept a PR to reimplement it, based on the new loading="lazy" method?
